charts/enterprise/authelia/questions.yaml

```diff
@@ -773,7 +773,7 @@ questions:
                             description: "deny"
                     - variable: subject
                       label: "Subject"
-                      description: "defines the subject to apply authorizations to. This parameter is optional and matching any user if not provided"
+                      description: "This criteria matches identifying characteristics about the subject. Currently this is either user or groups the user belongs to. This allows you to effectively control exactly what each user is authorized to access or to specifically require two-factor authentication to specific users. Subjects are prefixed with either user: or group: to identify which part of the identity to check."
```

Comment:
Please use multiline yaml (`key: |`) strings, don't have that long single lines.

Reply:
I hope I did that correctly.  I wasn't sure if I used the right syntax or put the line breaks in the right spot.